modules/cli/src/main/scala/scala/cli/exportCmd/MavenProjectDescriptor.scala

```diff
@@ -206,8 +191,7 @@ final case class MavenProjectDescriptor(
     scalaVersion: String
   ): MavenPlugin = {
 
-    val scalaVersionNode = buildNode("scalaVersion", scalaVersion)
-    val javacOptionsElem = {
+    {
```

Comment:
is this whole block needed at all? javacOptionsElem seems like it should probably be added to export somewhere. Might be an earlier bug -> maybe an issue? Do we properly export javacArgs?